docs/index.md

```diff
@@ -6,7 +6,7 @@ and execute them with the command line interface to generate **datasets with one
 ehrQL allows researchers to access data sources from primary and secondary care,
 as well as from organisations such as the Office for National Statistics (ONS).
 
-!!! info "Why did we create a new query language for healthcare research?  Read more in [this blog post](https://www.bennett.ox.ac.uk/blog/2023/09/why-ehrql/)."
+You can find out [why researchers should use ehrQL](https://www.bennett.ox.ac.uk/blog/2023/09/why-ehrql/) on the Bennett Institute blog.
```

Comment:
As well as being a good idea, it's also written in one of our organisation policy documents that we should always do this.

Reply:
```suggestion
You can find out about ehrQL's [philosophy and design goals](https://www.bennett.ox.ac.uk/blog/2023/09/why-ehrql/) on the Bennett Institute blog.
```

Reply:
My reasoning is that _[Link to answer]_ leads to _Click here_ and _Click here_ is undesirable. However, avoiding _Click here_ is cumbersome, hence the rewrite.

Let's think about the question. What kind of reader would ask, "Why did you decide to create a new language?" What kind of reader would ask, "What are ehrQL's philosophy and design goals?" I'd argue that this section of the docs should be pitched to the second kind of reader, rather than the first.

Reply:
Suggestion accepted 🙂